docs/tools/mbed_targets.md

```diff
@@ -186,6 +186,10 @@ The list in `device_has` defines what hardware a device has.
 
 Mbed, libraries and application source code can then select different implementations of drivers based on hardware availability; selectively compile drivers for existing hardware only; or run only the tests that apply to a particular platform. The values in `device_has` are available in C, C++ and assembly language as `DEVICE_` prefixed macros.
 
+#### `is_disk_virtual`
+
+When `is_disk_virtual` is enabled, there is a certain amount of delay added after flashing firmware binary to make sure the mount was correct. This field is not used anymore. Daplink uses mbedls to make sure the mount was correct before re-flashing it again.
```

Comment:
> Daplink uses mbedls to make sure the mount was correct before re-flashing it again

This is not correct. You can probably replace "Daplink" with "The Mbed test tools".

We should probably refer to "Mbed LS" in a consistent fashion. I would stick with the "Mbed LS" styling.